### DISCUSSION TITLE
How does the Typst company plan to make money?

### DISCUSSION THREAD

**Discussion by IllustratedMan-code** [typst/typst#3995] [Q&A]
This is sort of a question about the long-term viability of Typst (the company). Since the compiler is free and open source, I assume the monetization strategy for the "business" of Typst is going to be through the web app. My question is essentially: "What form will that monetization take?".   

Will the web app eventually be entirely subscription based (no free tier), or will the strategy be more like [OverLeaf's tier based model](https://www.overleaf.com/user/subscription/plans) where some features are behind a paywall? If this is the case, which features are planned to be paid eventually? Will the extensions always be free, or will there be some sort of paid extension market (similar to many game engines)?

I would just love some clarification, as my willingness to recommend the web app will most likely depend on the monetization strategy. Also, just wanted to say, the web app is great! I and 4 collaborators were able to work on the document simultaneously with little to no problems with latency. We were just talking about how much better the live collaboration is compared to Microsoft word documents.

**Comment by Enivex** [comment]
They've answered this in the past. There'll be a free tier similar to what exists today, but a paid tier will soon launch on top of that.

**Reply by IllustratedMan-code** [reply]
Any idea which features will be paid?

**Reply by blackjyn** [reply]
can you refer the link for that statement?

**Reply by Enivex** [reply]
> Any idea which features will be paid?

It's a good bet that most features not currently present will be paid.

**Reply by Enivex** [reply]
> can you refer the link for that statement?

I don't have a link, because the most up to date info was given in a recent community call on Discord

**Comment by IllustratedMan-code** [comment]
Now that typst (the web app) has a paid tier it seems pretty clear how they are making money, so I'll go ahead and close this discussion.
